src/rules/noDeclareMutable.ts

```diff
@@ -21,7 +22,14 @@ export class NoDeclareMutableError implements SourceError {
   }
 
   public elaborate() {
-    return this.explain()
+    const name = (this.node.declarations[0].id as es.Identifier).name
+    let value = astr.generate(this.node.declarations[0].init)
+
+    if (value.length >= 15) {
```

Comment:
Why does only this particular rule have an arbitrary max length of 15 when reproducing code? I think it's better to be consistent and apply this rule to all other errors or to not have it at all.

Reply:
I was thinking mainly because it has the potential to give you an entire function definition expression, which might lead to a messy error log. There's not as much of a need to have this in most other error messages, since many of them only have to deal with printing names. That said, I suppose the limit could be increased by quite a bit. I personally don't think it's good to not have a limit, though. Thoughts?

Reply:
Is the "code" nicely formatted? If yes, and if "verbose" is turned on, and if we can scroll, I don't see much harm. If in a particular situation, there is "too much information", the user can turn off "verbose". 

Having no limits: In this case we are talking about the programs that the user wrote. In the big scheme of things, programs are naturally finite, and in practice rather small, so there might be not much actual harm in having no limit (if the program is nicely presented).

Reply:
Fair point regarding disabling verbose; I'll remove the limit.